source/rust_verify/src/lifetime_generate.rs

```diff
@@ -1398,6 +1399,10 @@ fn erase_fn_common<'tcx>(
     } else {
         def_id_to_vir_path(ctxt.tcx, id)
     };
+    let is_verus_spec = path.segments.last().expect("segment.last").starts_with(VERUS_SPEC);
```

Comment:
Is it correct that the reason we rely on the name here is that the synthetic VERUS_SPEC methods do not appear in VIR, and are not otherwise tracked?

Reply:
Yes, the `VERUS_SPEC__f` trait methods are not considered real methods in VIR (I just added a test for this, confirming that programs can't call them directly) but are rather considered more like attributes on the `f` methods.